spread.yaml

```diff
@@ -408,7 +416,7 @@ suites:
 
     tests/unit/:
         summary: Suite to run unit tests (non-go and different go runtimes)
-        systems: [-ubuntu-core-16-*, -fedora-*]
+        systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*]
```

Comment:
Can you please document those too? I thought that we can run unit tests on openSUSE now, is this not the case?